examples/nextjs-comments-emails/src/app/page.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { useMemo } from "react";
+import { useSearchParams } from "next/navigation";
+import { RoomProvider, ThreadMetadata, useThreads } from "../liveblocks.config";
+import { Loading } from "../components/Loading";
+import { Composer, Thread } from "@liveblocks/react-comments";
+import { ClientSideSuspense } from "@liveblocks/react";
+import { ErrorBoundary } from "react-error-boundary";
+
+/**
+ * Displays a list of threads, along with a composer for creating
+ * new threads.
+ */
+
+function Example({ roomUrl }: { roomUrl: string }) {
+  const { threads } = useThreads();
+
+  return (
+    <main>
+      {threads.map((thread) => (
+        <Thread key={thread.id} thread={thread} className="thread" />
+      ))}
+      <Composer<ThreadMetadata>
+        className="composer"
+        // Add link to the room for email notifications
+        metadata={{ url: roomUrl }}
```

Comment:
I don't think we should use metadata for this as it's not realistic (users shouldn't have to do this). You should have access to the room ID on the webhook side?

Reply:
yes, the entire URL can be constructed from the webhook event. 

For the thread events: `projectId`, `roomId`, `threadId` are available
For comment events: the above and `commentId` are available

Reply:
@FlowFlorent Could you tweak the example to stop using thread metadata but `roomId` from the webhook events instead to build links within the emails?

Reply:
Yeah on it!

Reply:
Adjusted